tests/cpython-tests/Dockerfile3.9

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04
+WORKDIR /
+
+RUN apt update && apt install -y build-essential libssl-dev zlib1g-dev libncurses5-dev libncursesw5-dev \
+        libreadline-dev libsqlite3-dev libgdbm-dev libdb5.3-dev libbz2-dev \
+        libexpat1-dev liblzma-dev libffi-dev git
+
+RUN git clone -b v3.9.7 https://github.com/python/cpython
```

Comment:
If only the version string is different, would recommend using the same dockerfile and adding version as a build time argument - https://docs.docker.com/engine/reference/builder/#arg